templates/primary-secondary/primary-secondary.js

```diff
@@ -974,7 +1037,7 @@ class TemplatePrimarySecondary extends FocusVisiblePolyfillMixin(RtlMixin(Locali
 								</d2l-icon-custom>
 							</div>
 							<div class="d2l-template-primary-secondary-divider-handle-mobile">
-								${size === 0 ? html`<d2l-icon icon="tier1:chevron-up"></d2l-icon>` : html`<d2l-icon icon="tier1:chevron-down"></d2l-icon>`}
+								<d2l-icon icon=${size === 0 ? 'tier1:chevron-up' : 'tier1:chevron-down'}></d2l-icon>
```

Comment:
Since we added support for scrolling using the divider in mobile view on a touch screen, the resizing was being cancelled due to the `d2l-icon` DOM element being swapped out. This makes sure the icon changes but the DOM element is kept the same.